site/src/pages/components/accordion.explainer.mdx

```diff
@@ -172,7 +198,9 @@ then that would favor a design that separates the ideas
 of connecting the `details` elements into a group
 and making the group exclusive.
 
-## Possibly-resolved design questions
+In the [discussion on 2023 April 27](https://github.com/openui/open-ui/issues/725#issuecomment-1525924478),
+no use cases for a platform feature for connecting a non-exclusive accordion
+were raised.
```

Comment:
...other than the potential keyboard interaction pattern issue raised above?

Reply:
I added a caveat pointing to the previous issue, although I think my impression of the discussion of that issue was perhaps of a firmer conclusion to it than your impression.